omaha_server/omaha/tasks.py

```diff
@@ -39,14 +37,11 @@
 from sparkle.models import SparkleVersion
 from crash.models import Crash, Symbols
 from feedback.models import Feedback
-from sentry_sdk import capture_message
-
-logger = logging.getLogger(__name__)
 
 
 @app.task(ignore_result=True)
 def collect_statistics(request, ip=None):
-    statistics.collect_statistics(parse_request(request), ip=ip)
+    statistics.collect_statistics(parse_request(bytes(bytearray(request, encoding='utf-8'))), ip=ip)
```

Comment:
https://sentry.aws.dwos.com/dentalwings/omaha_server/issues/231/ something is still wrong here, the collect_statistics isn't able to get all requested data from the request :-(